tools/archetype/binding/src/main/resources/archetype-resources/src/main/feature/feature.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<features name="${rootArtifactId}-${project.version}" xmlns="http://karaf.apache.org/xmlns/features/v1.4.0">
```

Comment:
I didn’t test it: are you sure that only ${rootArtifactId} is replaced and ${project.version} stays in place? If so, LGTM.

Reply:
No, I am not sure.

Reply:
Replace goes well. `${project.version}` stays in place.